apps/condo/domains/marketplace/access/MarketSetting.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Generated by `createschema marketplace.MarketSetting 'organization:Relationship:Organization:CASCADE; residentAllowedPaymentTypes:Select:cash, online;'`
+ */
+
+const { compact, get, uniq } = require('lodash')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { find, getById } = require('@open-condo/keystone/schema')
+
+const { getEmployedOrRelatedOrganizationsByPermissions, checkPermissionsInEmployedOrRelatedOrganizations } = require('@condo/domains/organization/utils/accessSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canReadMarketSettings ({ authentication: { item: user }, context }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type === RESIDENT) {
+        const userResidents = await find('Resident', { user: { id: user.id }, deletedAt: null })
```

Comment:
<img width="879" alt="image" src="https://github.com/user-attachments/assets/430fe910-bb0e-45bb-8e45-4fb0794b2353">
this check is not needed. Since a deleted user will be cut off earlier